### PR TITLE
ICDS Dashboard: 6 digit Alphanumeric captcha

### PR DESCRIPTION
**Pull request by saiprakash-v**
Ticket: https://dimagi-dev.atlassian.net/browse/ICDS-1762
##### SUMMARY
Overriding the library challenge function with a custom function to generate 6 digit alphanumeric captcha

References:
https://django-simple-captcha.readthedocs.io/en/latest/advanced.html#roll-your-own
https://django-simple-captcha.readthedocs.io/en/latest/advanced.html#captcha-length

### CODE REVIEW

File: settingshelper.py [hunk 1]
```diff
@@ -282,3 +284,10 @@ def get_git_commit(base_dir):
         return out.strip().decode('ascii')
     except OSError:
         pass
+
+
+# Reference: https://django-simple-captcha.readthedocs.io/en/latest/advanced.html#roll-your-own
+def random_alphanumeric_challenge():
```
Comment (mkangia) [inline settingshelper.py:290]: that is great, its nice how straight forward this was.
Any reason you have taken uppercase and digits as opposed to just digits?

**Review by mkangia — CHANGES_REQUESTED** [review]
Can we put this in commcare-icds repo?

File: settingshelper.py (hunk 1 above)
Comment (saiprakash-v) [reply settingshelper.py:290]: Requirement by AKS was 6 digit Alphanumeric captcha. So I had to take both

**Comment by saiprakash-v** [comment]
> Can we put this in commcare-icds repo?

Move random captcha generator to commcare-icds repo?

**Comment by ShubhamBansal1997** [comment]
> Can we put this in commcare-icds repo?

Sorry to jump in between, I guess https://github.com/dimagi/icds-commcare-environments/blob/master/environments/icds-cas/public.yml  that would the best place to put the `settings.py` change for `icds-cas` and `icds-staging` resp.

**Comment by mkangia** [comment]
> Move random captcha generator to commcare-icds repo?

Actually everything @saiprakash-v since its all is in settings, we should be able to move it to commcare-icds repo.

**Pull request closed without merging**